docs/source/reading_writing.rst

```diff
@@ -0,0 +1,112 @@
+*******************
+Reading and writing
+*******************
+
+
+.. _reading chains:
+
+Reading chain files from PolyChord, MultiNest, CosmoMC, or Cobaya
+=================================================================
+
+If you have finished nested sampling or MCMC runs from PolyChord, MultiNest,
+CosmoMC, or Cobaya, then you should be able to read in the chain files
```

Comment:
Maybe some external links to these codes?

Reply:
Done in https://github.com/williamjameshandley/anesthetic/pull/243/commits/13e031b8132914758df51bcb733ea690536f010a.